knimin/lib/data_access.py

```diff
@@ -799,9 +799,14 @@ def pulldown(self, barcodes, blanks):
             survey_md = [''.join(['sample_name\t', '\t'.join(headers)])]
             for barcode, shortnames_answers in sorted(bc_responses.items()):
                 barcodes_seen.add(barcode)
-                ordered_answers = [shortnames_answers[h] for h in headers]
                 oa_hold = [barcode]
-                for x in ordered_answers:
+                for h in headers:
+                    # Take care of retired questions not having an answer
+                    try:
```

Comment:
`x` is a bad variable name

why not `answer = shortnames_answers.get(h, 'Unspecified')`?